buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```
@@ -24,7 +24,7 @@ java {
 
 checkstyle {
   configDirectory.set(file("$rootDir/buildscripts/"))
-  toolVersion = "8.12"
+  toolVersion = "10.20.1"
```

Well that didn't work.
```suggestion
  toolVersion = "8.45.1"
```